rules/aws_cloudtrail_rules/aws_user_login_profile_modified.py

```diff
@@ -2,15 +2,14 @@
 
 
 def rule(event):
-    if (
+    return (
         event.get("eventSource", "") == "iam.amazonaws.com"
         and event.get("eventName", "") == "UpdateLoginProfile"
-    ):
-        return (
-            deep_get(event, "requestParameters", "userName")
-            != deep_get(event, "userIdentity", "arn").split("/")[-1]
+        and (deep_get(event, "requestParameters", "passwordResetRequired", default="") is False)
```

Comment:
this is functional, and it surfaces a non-fatal SyntaxWarning on my local 
![Screen Shot 2022-11-01 at 15 13 15](https://user-images.githubusercontent.com/932424/199351952-bbd413ea-effa-42ea-a4a5-4226be8cb0a7.png)

I suspect it may be more appropriate to default to False

Reply:
I initially had `== False` but the black linter doesn't like that and wanted `is False`. 
I wanted to explicitly add the comparison just for readability sake so it's more clear what is expected but I could  remove it so the linters don't complain.